pkg/client/connector/auth/login.go

```diff
@@ -84,10 +89,12 @@ func NewLoginExecutor(
 		stdout:           stdout,
 		scout:            scout,
 
-		callbacks: make(chan oauth2Callback),
+		callbacks:    make(chan oauth2Callback),
+		refreshTimer: time.NewTimer(1 * time.Minute),
 	}
 	ret.oauth2ConfigMu.Lock()
 	ret.loginMu.Lock()
+	ret.resetRefreshTimer(0)
```

Comment:
Does this do anything? In the function we only do something if it's > 0 so this seems like a no-op whenever we call this with 0?

Reply:
It stops the timer when <= 0.  AFAICT there's not a way to create a timer in a stopped state, so I create it started with 1 minute to go, and then immediately stop it.

I should add comments about this.